pkg/scaffold/v2/crd/kustomizeconfig.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
+)
+
+var _ input.File = &KustomizeConfig{}
+
+// KustomizeConfig scaffolds the kustomizeconfig file in crd folder.
+type KustomizeConfig struct {
```

Comment:
we can put all `v2/crd/xxxx.go` scaffolders in same file `v2/crd/crd.go`

Reply:
Marking this comment as resolved per https://github.com/kubernetes-sigs/kubebuilder/pull/644#issuecomment-488759892.